app/javascript/packs/react-bootloader.jsx

```diff
@@ -41,15 +41,6 @@ const renderTooltips = (mappings) => {
       const tooltipElem = document.createElement('div')
       elem.insertAdjacentElement('afterend', tooltipElem)
 
-      // Link the tooltip element with the reference element
```

Comment:
I removed this because this is redundant code. `<Tooltip />` already adds the popper functionality.